src/category_theory/opposites.lean

```diff
@@ -9,47 +9,48 @@ import data.opposite
 
 universes v₁ v₂ u₁ u₂ -- morphism levels before object levels. See note [category_theory universes].
 
-namespace category_theory
 open opposite
 
 variables {C : Type u₁}
 
-section has_hom
+section quiver
 
-variables [has_hom.{v₁} C]
+variables [quiver.{v₁} C]
 
 /-- The hom types of the opposite of a category (or graph).
 
   As with the objects, we'll make this irreducible below.
   Use `f.op` and `f.unop` to convert between morphisms of C
   and morphisms of Cᵒᵖ.
 -/
-instance has_hom.opposite : has_hom Cᵒᵖ :=
+instance quiver.opposite : quiver Cᵒᵖ :=
```

Comment:
This looks like a duplicate of `quiver.op_quiver`?

Reply:
Thanks. Fixed.